lib/chef/application/client.rb

```diff
@@ -21,6 +21,7 @@
 require_relative "../handler/error_report"
 require_relative "../workstation_config_loader"
 require "uri" unless defined?(URI)
+require "chef-utils"
```

Comment:
Should I change this to the following?
```suggestion
require "chef-utils/dsl/windows"
require "chef-utils/dist"
```

Reply:
nope